hello_world_on_chain/contracts/User.sol

```diff
@@ -2,33 +2,20 @@
 pragma solidity ^0.8.10;
 
 import {TAParserLib} from "../lib/TAParserLib.sol";
-import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
```

Comment:
Example contract no longer needs to be ownable.  If we want to make a more complex user contract, we can still do that as a more "advanced usage section"

Reply:
updated docs.